can-simple-map.js

```diff
@@ -181,14 +202,17 @@ canReflect.assignSymbols(SimpleMap.prototype,{
 	},
 	"can.getKeyDependencies": function(key) {
 		return undefined;
-	},
+	}	
+};
 
-	//!steal-remove-start
-	"can.getName": function() {
+//!steal-remove-start
+if (process.env.NODE_ENV !== 'production') {
+	simpleMapProto[canSymbol.for("can.getName")] = function() {
```

Comment:
This canSymbol.for() isn't needed since `simpleMapProto` is passed into assignSymbols

Reply:
Thanks, it's fixed!